### ISSUE TITLE
question: secret storage

### ISSUE THREAD

**Issue by bkeroackdsc** [mailgun/vulcand#189]
Hi,

It says in the documentation that vulcand can transparently decrypt data stored in the backend, however it's not clear to me how this can be used to serve arbitrary encrypted data?

eg, if we had a number of encrypted keys in etcd (or served by the backend), how would we indicate to vulcand that they (and only they) require decryption?

**Comment by porjo** [comment]
Are you referring to [this](http://www.vulcanproxy.com/proxy.html#secret-storage)? AFAIK that is only for specific purposes used by Vulcand, not a general purpose service.

**Comment by bkeroackdsc** [comment]
@porjo Yes that was my impression as well, that it's only for certificates and keys specifically, but the wording is ambiguous. If it isn't a general purpose facility the docs should be explicit about this.
